jupyter-R.el

```diff
@@ -0,0 +1,111 @@
+;;; jupyter-R.el --- Jupyter support for R -*- lexical-binding: t -*-
+
+;; Copyright (C) 2019 Nathaniel Nicandro
+
+;; Author: Jack Kamm <jackkamm@gmail.com>, Nathaniel Nicandro <nathanielnicandro@gmail.com>
+
+;; This program is free software; you can redistribute it and/or
+;; modify it under the terms of the GNU General Public License as
+;; published by the Free Software Foundation; either version 2, or (at
+;; your option) any later version.
+
+;; This program is distributed in the hope that it will be useful, but
+;; WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+;; General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with GNU Emacs; see the file COPYING.  If not, write to the
+;; Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+;; Boston, MA 02111-1307, USA.
+
+;;; Commentary:
+
+;; Support methods for integration with R.
+
+;;; Code:
+
+(require 'jupyter-repl)
+
+(cl-defmethod jupyter-repl-initialize-fontification (&context (jupyter-lang R))
+  (if (featurep 'ess)
+      (setq-local ess-font-lock-keywords 'ess-R-font-lock-keywords))
+  (cl-call-next-method))
+
+(cl-defmethod jupyter-org-result ((_mime (eql :text/html)) _params data
+                                  &context (jupyter-lang R)
+                                  &optional _metadata)
```

Comment:
You should change `_metadata` and `_params` to just `metadata` and `params` since you use the arguments in the method body. The underscore is used to indicate that an argument is **not** used inside the method and so that the byte compiler does not complain about unused variables.

Reply:
Done